PopPUNK/network.py

```diff
@@ -1315,5 +1444,38 @@ def save_network(G, prefix = None, suffix = None, use_gpu = False):
         G.to_pandas_edgelist().to_csv(file_name + '.csv.gz',
                 compression='gzip', index = False)
     else:
-        G.save(file_name + '.gt',
-                fmt = 'gt')
+        if use_graphml:
+            G.save(file_name + '.graphml',
+                    fmt = 'graphml')
+        else:
+            G.save(file_name + '.gt',
+                    fmt = 'gt')
+
+def cugraph_to_graph_tool(G, rlist = None):
+    """Save a network to disc
+
+    Args:
+       G (cugraph network)
+         Cugraph network
+       rlist (list)
+         List of sequence names
+           
+    Returns:
+      G (graph-tool network)
+          Graph tool network
+    """
+    edge_df = G.view_edge_list()
+    edge_tuple = edge_df[['src', 'dst']].values
+    if 'weights' in edge_df.columns:
+        G = constructNetwork(rlist, rlist,
+                               edge_tuple,
+                               0, edge_list=True,
+                               weights=edge_df['weights'].values_host,
```

Comment:
Possible/easier to just set weights = None rather than writing call out twice?

Reply:
Good point, done in e32b75c.